bundles/org.openhab.binding.pjlinkdevice/src/main/java/org/openhab/binding/pjlinkdevice/internal/discovery/AbstractDiscoveryParticipant.java

```diff
@@ -63,7 +60,7 @@ protected ExecutorService getExecutorService() {
     @Override
     protected void startScan() {
         logger.trace("PJLinkProjectorDiscoveryParticipant startScan");
-        Set<InetAddress> addressesToScan = generateAddressesToScan();
+        List<InetAddress> addressesToScan = NetUtil.getFullRangeOfAddressesToScan();
```

Comment:
Previous code was ignoring loopback interface and any interface that is not up.
New code does not ?

Reply:
`NetUtil.getFullRangeOfAddressesToScan()`  calls `NetUtil.getAllInterfaceAddresses()` and that method filters:

https://github.com/openhab/openhab-core/blob/389f6a34343a723442829b7d9841240adc2f2a9c/bundles/org.openhab.core/src/main/java/org/openhab/core/net/NetUtil.java#L344